assets/js/admin/admin-scripts.js

```diff
@@ -193,6 +193,7 @@ var give_setting_edit = false;
 			inherit_select_classes: true,
 			placeholder_text_single: give_vars.one_option,
 			placeholder_text_multiple: give_vars.one_or_more_option,
+			width: '100%'
```

Comment:
@mehul0810 do not set all chosen fields width to `100%`. It will break many things.